PROJECT_LIFECYCLE.md

```diff
@@ -96,6 +96,13 @@ moving to a different stage.
 Due to the nature of sandbox projects (eg. experimentation), project can be moved to archive stage at any point in
 time if the maintainers deem it necessary.
 
+You should use the ![Sandbox badge](https://img.shields.io/badge/Stage-Sandbox-yellow.svg) badge to show the project stage.
```

Comment:
It would help if this badge linked to the correct internal document instead:
`[![Graduated badge](https://img.shields.io/badge/Stage-Graduated-green)](https://github.com/eiffel-community/community/blob/master/PROJECT_LIFECYCLE.md#stage-graduated)
`as is done here: https://github.com/eiffel-community/eiffel/pull/247/files
The below specified markdown didn't help me too much to get the syntax correct, as it is restructured text and not markdown.

Reply:
Fixed in 08037d6